draft-ietf-scitt-architecture.md

```diff
@@ -234,7 +234,7 @@ Registration Policy:
 
 Relying Party:
 
-: a relying party depends on Signed or Transparent Statements to verify an Artifact.
+: a Relying Parties consumes Signed Statements, verifying their proofs and inspecting the Statement payload, either before using corresponding Artifacts, or later to audit an Artifact's provenance on the supply chain.
```

Comment:
```suggestion
: a Relying Parties consumes Transparent Statements, verifying their proofs and inspecting the Statement payload, either before using corresponding Artifacts, or later to audit an Artifact's provenance on the supply chain.
```

Reply:
Note a Relying Party gets what is returned by the Transparency Service

Reply:
goo catch